addons/mrp/tests/test_order.py

```diff
@@ -4676,6 +4676,42 @@ def test_json_popover_with_workorder_dependence(self):
         wos_to_set.write({'date_start': date_start, 'date_finished': date_finished})
         self.assertTrue(mo.workorder_ids[-1].show_json_popover)
 
+    def test_update_mo_from_bom_forecast(self):
```

Comment:
move to `test_warehouse_multistep_manufacturing` and also add in byproducts to the test